lib/fencing/st_output.c

```diff
@@ -139,20 +140,23 @@ stonith__failed_history(pcmk__output_t *out, va_list args) {
         PCMK__OUTPUT_LIST_HEADER(out, print_spacer, rc, "Failed Fencing Actions");
         out->message(out, "stonith-event", hp,
                      pcmk_all_flags_set(section_opts, pcmk_section_fencing_all),
-                     stonith__later_succeeded(hp, history), true);
+                     stonith__later_succeeded(hp, history),
+                     pcmk_is_set(show_opts, pcmk_show_failed_detail));
         out->increment_list(out);
```

Comment:
hmm ... going for a flag-field here would it make sense to pass the flags through to the bottom for the case we want to add a flag that does additional modification to the history-string?
If some info is just available somewhere in the middle we could still modify the flags on the way.
Of course we don't know if we'll ever need it ...

Reply:
Oops no you didn't - so question above is still valid ...

Reply:
Yep, every time I went back to that one I thought about it, I should have done it :)